portal/views/auth.py

```diff
@@ -521,14 +523,15 @@ def client():
     db.session.add(client)
     db.session.commit()
     auditable_event("added intervention/client {}".format(
-        client), user_id=user.id)
+        client), user_id=user.id, subject_id=client.user_id)
```

Comment:
@pbugni I agree that here (line 524), it makes sense just to set `subject_id=user.id` (especially since that's actually the same value as `client.user.id`). But there are some spots where I think it may make more sense to use `client.user_id`? For example, check out line 631. The editing user is not the same one as the user being edited (we even check to see if `current_user()` has permission to edit `client.user_id` above it). Would it not make more sense that the subject of the audit is the `client.user_id`, rather than the `current_user()` doing the editing?